MedicalDocManagment.WebUI/app/admin/models/usersmodel.ts

```diff
@@ -1,12 +1,10 @@
-﻿import 'rxjs/add/operator/map';
-
-import UserModel from '../models/usermodel';
+﻿import UserModel from '../models/usermodel';
 
 export default class UsersModel extends Array<UserModel> {
-    constructor(jsonobject?) {
+    constructor(jsonObject?) {
         super();
-        if (jsonobject) {
-            jsonobject.map((user) => { this.push(new UserModel(user)); });
+        if (jsonObject) {
+            jsonObject.forEach((user) => { this.push(new UserModel(user)); });
```

Comment:
Yes, it works on my side — I checked features "users list" and "search user" and they are working.
Where exactly did you have troubles?